pkg/controller/status/status.go

```diff
@@ -219,6 +230,23 @@ func (c *Controller) merge(existing *configv1.ClusterOperator) *configv1.Cluster
 				Status: configv1.ConditionFalse,
 			})
 		}
+
+		if len(uploadErrorReason) > 0 {
+		    klog.Errorf("=== Uplaoderror")
+			setOperatorStatusCondition(&existing.Status.Conditions, configv1.ClusterOperatorStatusCondition{
```

Comment:
Someone has to reset this condition when uploading is not false.  You'd have to do that in both halves of this else block

Reply:
added `removeOperatorStatusCondition` in the else, well as in `isInitializing` case, if that's what this ask is about.